### PR TITLE
added was_removed analytics to closing edit set

### PR DESCRIPTION
**Pull request by Malekai**

### CODE REVIEW

File: app/features/analysis/edit_set/OneRMEditSetActions.js [hunk 1]
```diff
@@ -75,5 +76,6 @@ const logCloseEditSetAnalytics = (state) => {
         did_change_rpe: didChangeRPE,
         did_change_tags: didChangeTags,
         did_change_reps: didChangeReps,
+        was_removed: wasRemoved,
```
Comment (Malekai) [inline app/features/analysis/edit_set/OneRMEditSetActions.js]: Why are we calling it was_removed in our analytics? wouldn't was_error make more sense?

Comment (JonnyLinja) [reply app/features/analysis/edit_set/OneRMEditSetActions.js]: - [ ] make it was_error following the v3 analytics doc

File: app/redux/reducers/AnalysisReducer.js [hunk 2]
```diff
@@ -94,6 +94,7 @@ const defaultState = {
     origWeight: null,
     origMetric: null,
     origTags: [],
+    error: false,
```
Comment (JonnyLinja) [inline app/redux/reducers/AnalysisReducer.js]: - [ ] rename to wasError

File: app/math/OneRMCalculator.js [hunk 3]
```diff
@@ -117,7 +117,7 @@ export const calculate1RM = (exercise, tagsToInclude, tagsToExclude, daysRange,
     };
 };
 
-const convertToChartData = (array, metric, displayMarker=true) => {
+const convertToChartData = (array, metric, displayMarker=true, error=false) => {
```
Comment (JonnyLinja) [inline app/math/OneRMCalculator.js]: - [ ] rename to wasError

File: app/redux/selectors/AnalysisSelectors.js [hunk 4]
```diff
@@ -105,6 +105,8 @@ export const getOrigTags = (state) => stateRoot(state).origTags;
 
 export const getOrigDeletedFlag = (state) => stateRoot(state).origDeletedFlag;
 
+export const getWasError = (state, set) => stateRoot(state).wasError;
```
Comment (JonnyLinja) [inline app/redux/selectors/AnalysisSelectors.js]: - [ ] Why do you pass in set?

File: app/features/analysis/edit_set/OneRMEditSetActions.js [hunk 5]
```diff
@@ -67,6 +67,7 @@ const logCloseEditSetAnalytics = (state) => {
     const didChangeReps = AnalysisSelectors.getDidUpdateReps(state, set);
     const didDeleteSet = AnalysisSelectors.getDidDeleteSet(state, set);
     const didRestoreSet = AnalysisSelectors.getDidRestoreSet(state, set);
+    const wasError = AnalysisSelectors.getWasError(state)
```
Comment (JonnyLinja) [inline app/features/analysis/edit_set/OneRMEditSetActions.js]: - [ ] semicolon

**Pull request merged**